deploy/example/storageclass-smb.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: smb
 provisioner: smb.csi.k8s.io
 parameters:
-  source: "//smb-server.default.svc.cluster.local/share"
```

Comment:
this storage class should only works for Linux since `smb-server.default.svc.cluster.local` could not be recognized by csi-proxy.exe, I think it's better add a notes here.